### DISCUSSION TITLE
hey, I need some knowledge about developing a database plugin, can you help me out ?

### DISCUSSION THREAD

**Discussion by kurt-steiner** [microsoft/vscode-discussions#1222] [Extension Development QnA] [answered]
hey, guys, I am a noob of plugin development, and now I want to develop a plugin for project-based-learning, here is my idea
1. I want to develop a postgresql database plugin
2. it is very simple, just show the record of specify data table
3. and now I want to use webview to decorate it, and I am using vue3 + typescript

so, based on these ideas, can you give me a roadmap or suggestions ?

**Comment by isidorn** [comment, marked as answer]
Here's the existing postgressql extensions that is open source https://github.com/microsoft/vscode-postgresql
You can look at it for inspiration
